okapi-core/src/test/java/org/folio/okapi/ProxyTest.java

```diff
@@ -4537,6 +4537,83 @@ private void setupBasicModule(String tenant, String moduleId, String tenantPermi
         .then().statusCode(200).log().ifValidationFails();
   }
 
+  @Test
+  public void testObsoleteModules2(TestContext context) {
+    RestAssuredClient c;
+
+    given()
+        .header("Content-Type", "application/json")
+        .body("{}").post("/_/proxy/cleanup/modules?saveSnapshots=2")
+        .then().statusCode(400)
+        .body(is("Missing value for parameter 'saveReleases'"));
+    given()
+        .header("Content-Type", "application/json")
+        .body("{}").post("/_/proxy/cleanup/modules?saveReleases=1")
+        .then().statusCode(400)
+        .body(is("Missing value for parameter 'saveSnapshots'"));
+    c = api.createRestAssured3();
+    c.given()
+        .header("Content-Type", "application/json")
+        .body("{}").post("/_/proxy/cleanup/modules?saveReleases=1&saveSnapshots=2")
+        .then().statusCode(204);
+    Assert.assertTrue("raml: " + c.getLastReport().toString(),
+        c.getLastReport().isEmpty());
```

Comment:
```suggestion
    api.failFast().createRestAssured3()
    .given()
        .header("Content-Type", "application/json")
        .body("{}").post("/_/proxy/cleanup/modules?saveReleases=1&saveSnapshots=2")
        .then().statusCode(204);
```
The Assert can be replaced by failFast().